scss/_progress.scss

```diff
@@ -1,32 +1,37 @@
+// Progress animations
 @keyframes progress-bar-stripes {
   from { background-position: $progress-height 0; }
   to { background-position: 0 0; }
 }
 
+// Basic progress bar
 .progress {
   display: flex;
+  align-items: center;
+  height: $progress-height;
   overflow: hidden; // force rounded corners by cropping it
   font-size: $progress-font-size;
   line-height: $progress-height;
```

Comment:
Height is good to stay here, but I meant we can probably remove the `line-height`.